feature-libs/cart/main/root/facade/multi-cart.facade.ts

```diff
@@ -80,7 +80,7 @@ export abstract class MultiCartFacade {
     extraData?: {
       active?: boolean;
     };
-  }): Observable<StateUtils.ProcessesLoaderState<Cart | undefined>>;
```

Comment:
Didn't UI components need the state to display spinners ans react to errors?

Reply:
When we create a cart, we first add a temp cart in state, which has `loading=true`, then after we get response from backend, we add a new cart to state, and remove the temp cart. So, before, it just returns the temp cart with state, but this temp cart will be removed soon (the new cart only has state of `loaded=true`). For the spinner, if there is no emit, then UI can display the spinner.